tests/cockpit-tests

```diff
@@ -37,7 +30,7 @@ if sudo -n ip address show dev cockpit1 >/dev/null 2>/dev/null; then
 fi
 
 if [ ! -d cockpit ]; then
-    sudo chown -R user /build
+    sudo chown -R user .
```

Comment:
Not really, `./` is exactly the same as `.`. It's rather blunt, but oh well,  this is all inside a container, so the risk of breaking something permanently is low.